RadixWallet/Core/SharedModels/P2P/Dapp/Models/Interaction/Requests/PersonaDataRequestItem.swift

```diff
@@ -83,35 +79,3 @@ private extension [PersonaData.Entry] {
 		try .init(validating: map { try $0.extract() })
 	}
 }
-
-extension PersonaData {
-	public func responseValidation(for request: P2P.Dapp.Request.PersonaDataRequestItem) -> P2P.Dapp.Request.RequestValidation {
```

Comment:
moved to  https://github.com/radixdlt/babylon-wallet-ios/blob/sargon_profile_integration_stage_1_batch_1/RadixWallet/ProfileSargonMigration/Stage2/Atoms/Stage2MigrateToSargpn%2BPersonaData.swift#L5